src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,13 @@
 package org.example;
 
+import com.opencsv.exceptions.CsvValidationException;
+import mandel.dictionary.EnglishDictionary;
+
+import java.io.IOException;
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public static void main(String[] args) throws CsvValidationException, IOException {
+        EnglishDictionary dict = new EnglishDictionary();
+        System.out.println(dict.getDefinition("Aboriginal"));
```

Comment:
Why?

Reply:
I just wanted to test it in main to see what would print so I picked a random word with multiple translations.

Reply:
That's what the unit test is for. Get out of the habit of doing this.